### PR TITLE
[ADD] doc: gantt view cell_create attribute.

### PR DESCRIPTION
**Pull request by sga-odoo**
**LINKS**
Task: 2081187
Closes: https://github.com/odoo/odoo/pull/46205

### CODE REVIEW

File: doc/reference/views.rst [hunk 1]
```diff
@@ -939,12 +939,14 @@ take the following attributes:
   name of the field that describes if the task has to be excluded
   from the consolidation
   if set to true it displays a striped zone in the consolidation line
-``create``, ``edit``, ``delete``, ``plan``
+``create``, ``cell_create``, ``edit``, ``delete``, ``plan``
     allows *dis*\ abling the corresponding action in the view by setting the
     corresponding attribute to ``false``.
 
     * ``create``: If enabled, a "**+**" button will be displayed
```
Comment (aab-odoo) [inline doc/reference/views.rst]:
```
If enabled, an ``Add`` button will be available in the control panel to create records.
```

File: doc/reference/views.rst [hunk 2]
```diff
@@ -939,12 +939,14 @@ take the following attributes:
   name of the field that describes if the task has to be excluded
   from the consolidation
   if set to true it displays a striped zone in the consolidation line
-``create``, ``edit``, ``delete``, ``plan``
+``create``, ``cell_create``, ``edit``, ``delete``, ``plan``
     allows *dis*\ abling the corresponding action in the view by setting the
     corresponding attribute to ``false``.
```
Comment (aab-odoo) [inline doc/reference/views.rst]:
```
allows *dis*\ abling the corresponding action in the view by setting the corresponding attribute to ``false`` (default: ``true``).
```

Comment (sga-odoo) [reply doc/reference/views.rst]: changes done!!

**Comment by aab-odoo** [comment]
@sga-odoo I slightly modified the text. Also, be careful with trailing spaces... I approve this PR, and I'll let @dbo-odoo r+ it alongside the enterprise one.

**Comment by dbo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**